packages/flutter_tools/bin/hack_script.sh

```diff
@@ -0,0 +1,7 @@
+eval "$(echo "static const int Moo = 88;" | xcrun clang -x c \
```

Comment:
What's the `eval` wrapper for?

Reply:
I copy pasted this code without really questioning it. I've cleaned it up to no longer require the bash script